qa/rpc-tests/maxuploadtarget.py

```diff
@@ -248,7 +247,7 @@ def run_test(self):
         #stop and start node 0 with 1MB maxuploadtarget, whitelist 127.0.0.1
         print "Restarting nodes with -whitelist=127.0.0.1"
         stop_node(self.nodes[0], 0)
-        self.nodes[0] = start_node(0, self.options.tmpdir, ["-debug", "-whitelist=127.0.0.1", "-maxuploadtarget=1", "-blockmaxsize=999000"])
+        self.nodes[0] = start_node(0, self.options.tmpdir, ["-debug", "-whitelist=127.0.0.1", "-maxuploadtarget=1", "-blockmaxsize=999000" , "-mocktime="+str(int(time.time()))])
```

Comment:
@MarcoFalke It works either way but I suppose for consistency it makes sense, I pushed up the change to maxuploadtarget.